internal/pkg/template/input.go

```diff
@@ -99,4 +112,25 @@ func checkTypeAgainstKind(value interface{}, kind string) error {
 	return nil
 }
 
+func validateAgainstRulesString(userInput interface{}, rules string, ctx context.Context) error {
+	validate := goValidator.New()
+	return validate.VarCtx(ctx, userInput, rules)
+}
+
+func checkPanic(fn func() error) (err, recovered interface{}) {
+	defer func() {
+		recovered = recover()
+	}()
+	return fn(), recovered
+}
+
+func (i Input) Validate(userInput interface{}, ctx context.Context) error {
+	err := checkTypeAgainstKind(userInput, i.Kind)
+	if err != nil {
```

Comment:
toto sa zvykne spojit do jedneho riadku, ak sa err pouziva iba tam
```
	if err := checkTypeAgainstKind(userInput, i.Kind); err != nil {
```

Vascinou to hlasi aj linter.

Reply:
👍